04.ls/ls.rb

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env ruby
+# frozen_string_literal: true
+
+COLUMNS = 3
+COLUMNS_DISTANCE = 5
+
+def main(file_names)
+  return exit if file_names.empty?
```

Comment:
exitは呼んだ瞬間にプログラムが終了するのでreturnする必要がないです。

```suggestion
  exit if file_names.empty?
```

ただし、exitは基本的に「プログラムの緊急停止ボタン」なので、「どうしても」というとき意外は使わない方がいいです。
ここはexitを使わずにただのreturnにすれば、処理を中断することができます。


```suggestion
  return if file_names.empty?
```

Reply:
exitを削除しました。